x-pack/test_serverless/functional/test_suites/common/management/index_management/enrich_policies.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const pageObjects = getPageObjects(['common', 'indexManagement', 'header', 'svlCommonPage']);
+  const log = getService('log');
+  const browser = getService('browser');
+  const security = getService('security');
+  const testSubjects = getService('testSubjects');
+  const es = getService('es');
+
+  const ENRICH_INDEX_NAME = 'test-policy-1';
+  const ENRICH_POLICY_NAME = 'test-policy-1';
+
+  describe('Enrich policies tab', function () {
+    // TimeoutError:  Waiting for element to be located By(css selector, [data-test-subj="kibana-chrome"])
+    this.tags(['failsOnMKI']);
```

Comment:
Is this a known error? Maybe we should open an issue to investigate and possibly address this?